package.json

```diff
@@ -28,13 +28,15 @@
     "gulp-rename": "2.0.0",
     "jose": "^4.11.2",
     "saml-idp": "^1.2.1",
-    "selenium-webdriver": "^4.0.0-alpha.7",
     "selfsigned": "^2.0.1",
     "typescript": "4.0.2"
   },
   "dependencies": {
     "@hapi/cryptiles": "5.0.0",
     "@hapi/wreck": "^17.1.0",
     "html-entities": "1.3.1"
+  },
+  "resolutions": {
+    "selenium-webdriver": "4.10.0"
```

Comment:
Nit: Was there any pkgs dpending on the selenium-webdriver with different versions? Just trying to figure out why we are using the selective dependencies resolution here.

Reply:
This is forcing to 4.10.0 due to a suspected bug in selenium. See the versions here: https://www.npmjs.com/package/selenium-webdriver?activeTab=versions

4.11.0 was published 3 days ago and 4.11.1 2 days ago. We can try removing this forced resolution on future versions of selenium. I'm going to look through the selenium issue list to look for any known issues: https://github.com/SeleniumHQ/selenium/issues

Reply:
I suspect that the error comes from this [PR in Selenium](https://github.com/SeleniumHQ/selenium/pull/12411/files), but I don't know enough about selenium to be able to determine why it produces the error we are seeing.